aframe.html

```diff
@@ -17,6 +17,8 @@
 AFRAME.registerComponent('console', {
   init: function () {
     let url = localStorage['url'] || 'http://gateway.local:8080';
+    if (!localStorage['url'])
+	    console.warn( "Gateway token unset. Visit your gateway Settings -> Developer -> Create local authorization" )
```

Comment:
ok, 
maybe move upper line and remove the ||

for indentation is there any rule ? I tend to use 2 spaces ?